cmake/std/atdm/cts1empire/all_supported_builds.sh

```diff
@@ -0,0 +1,9 @@
+
+export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-
+
+export ATDM_CONFIG_ALL_SUPPORTED_BUILDS=(
+  cts1-intel-18.0.2_openmpi-4.0.1_openmp_static_dbg
+  cts1-intel-18.0.2_openmpi-4.0.1_openmp_static_opt
+  cts1-intel-19.0.5_openmpi-4.0.1_openmp_static_dbg
+  cts1-intel-19.0.5_openmpi-4.0.1_openmp_static_opt
```

Comment:
Given that EMPIRE does not work with intel-19, we should likely delete these intel-19 builds.